pkg/securitycontextconstraints/sccadmission/admission.go

```diff
@@ -129,6 +134,13 @@ func (c *constraint) computeSecurityContext(ctx context.Context, a admission.Att
 	// get all constraints that are usable by the user
 	klog.V(4).Infof("getting security context constraints for pod %s (generate: %s) in namespace %s with user info %v", pod.Name, pod.GenerateName, a.GetNamespace(), a.GetUserInfo())
 
+	err := wait.PollImmediate(1*time.Second, 10*time.Second, func() (bool, error) {
```

Comment:
will this race with the overall admission timeout?

Reply:
> will this race with the overall admission timeout?

It's pretty far under it.  the admission timeout is higher.